bin/upgrade/DEV/upgrade.mongo.50.sh

```diff
@@ -45,13 +49,21 @@ if [ -x "$(command -v mongo)" ]; then
     fi
 
     if [ -f /etc/redhat-release ]; then
+        #backup of systemd unit file and mongod.conf file
+        \cp /usr/lib/systemd/system/mongod.service /usr/lib/systemd/system/mongod.service.bak
+        \cp -f /etc/mongod.conf /etc/mongod.conf.bak
         #uninstall mognodb
         yum erase -y mongodb-org mongodb-org-mongos mongodb-org-server mongodb-org-shell mongodb-org-tools
     fi
 
     if [ -f /etc/lsb-release ]; then
+        UBUNTU_YEAR="$(lsb_release -sr | cut -d '.' -f 1)";
+        if [ "$UBUNTU_YEAR" == "22" ]; then
+            sed -i "s/#\$nrconf{restart} = 'i';/\$nrconf{restart} = 'a';/" /etc/needrestart/needrestart.conf
+        fi
         #uninstall mognodb
-        apt-get remove -y mongodb-org mongodb-org-mongos mongodb-org-server mongodb-org-shell mongodb-org-tools
+        apt-get remove -y mongodb-org mongodb-org-mongos mongodb-org-server mongodb-org-shell mongodb-org-tools python3-apt
+        apt-get install python3-apt -y
```

Comment:
follow the convention and put `-y` after `install`

Reply:
made the changes according to convention